### PR TITLE
update to 60.2

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
sorry i'm not very smart, so I just copy pasted Anaconda's version into here using the online editor.
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
I guess the only difference at this point is MSVC2015 vs MSVC2017?

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

**Comment by isuruf** [comment]
Can we merge this given https://github.com/conda-forge/icu-feedstock/issues/21 ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,37 @@
-{% set version = "58.2" %}
+{% set version = "60.2" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I believe that only `aarch64` will use this version, unless we update the pin, so I'm OK merging this and we can slowly test it with the rest of the stack before bumping the pin everywhere.

**Pull request merged**